confirm/command.go

```diff
@@ -31,11 +31,11 @@ func (o Options) Run() error {
 		WithTheme(theme).
 		WithShowHelp(o.ShowHelp).
 		Run()
-
 	if err != nil {
-		if !o.errIsValidTimeout(err) {
-			return fmt.Errorf("unable to run confirm: %w", err)
+		if errors.Is(err, huh.ErrTimeout) {
+			return exit.NewTimeout(o.Timeout)
 		}
+		return err
```

Comment:
You are repeating this exact code in multiple places in multiple files

Why not adding a function

Something like

```go
if err != nil {
   return o.handleTimeout(err)
}
```

And then define handleTimeout method to return exit.NewTimeout on huh.Timeout

Reply:
each command has its own `Options`, would have to repeat anyway.

thought about moving it to `exit`, but feels weird